modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/notification/DDMFormEmailNotificationSender.java

```diff
@@ -500,14 +500,14 @@ protected String getViewFormEntriesURL(
 	protected String getViewFormURL(
 			DDMFormInstance ddmFormInstance,
 			DDMFormInstanceRecord ddmFormInstanceRecord,
-			ThemeDisplay themeDisplay)
+			ServiceContext serviceContext)
```

Comment:
@antonio91dev please order the parameters to be consistent with the other methods `getEmailBody`, `createTemplate`, `createMailMessage` and `populateParameters `.